15_perms_lab/permslab.txt

```diff
@@ -1,15 +1,43 @@
 1. Using the file structure in the current directory, write the commands that perform the following:
 
-  - change permission of file src/main.c to r--r--r-- using chmod letter notation
+  - change permission of file src/main.c to r--r--r-- 
+chmod u-w src/main.c
+
+using chmod letter notation
 
-  - change permission of file include/player.h to r-xrw--r- using chmod octal (numbers) notation
+  - change permission of file include/player.h to r-xrw--r-
+Note: "r" in my case is in the middle, not in the beginning 
+chmod u-w,q+xw include/player.h
+ 
+using chmod octal (numbers) notation
 
   - change permission of all files in Doc folder (without changing the permissions of Doc folder itself) to rw-------
 
-  - change permission of "lib" folder and all files and subdirectories inside it to rwxrwxrwx
+  -
+chmod 600 Doc/*
+
+ change permission of "lib" folder and all files and subdirectories inside it to rwxrwxrwx
 
-2. create a new directory named "test" and inside an empty file named "secret". Change permissions of the file "secret" to ---------. Can you delete the file secret? can you delete the folder test? why?
+
+
+chmod 777 -R lib
+
 
+2. 
+create a new directory named "test" and inside an empty file named "secret". 
+Change permissions of the file "secret" to ---------. Can you delete the file secret? can you delete the folder test? why?
+
+
+
+chmod 000 test/*
+
+Yes, I can delete the file and the test, im not sure why if basically it's untouchable 
+

```

Comment:
Yeah it was my mistake in the exercise. Your solution is good, but I fixed the exercise now so if you have time you can try the new version